apps/meteor/app/apps/server/bridges/livechat.ts

```diff
@@ -138,6 +138,27 @@ export class AppLivechatBridge extends LivechatBridge {
 		return true;
 	}
 
+	protected async findRoomsByAgentId(agentId: string, appId: string): Promise<ILivechatRoom[]> {
```

Comment:
I've just noticed that the implementation includes some filtering that is not clear by the method's name, i.e. `findRoomsByAgentId` does not convey that the method will filter by `{open: true}` - which the method `findOpenByAgent` makes more clear.

So my suggestion here would be to rename the methods in the Apps-Engine to something more similar to `findOpenRoomsByAgentId` and add some comments to the methods explaining what filters it will apply, so it is clearer from the Apps-Engine perspective.

Reply:
@d-gubert can you help me with the release for that change: https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/735

Reply:
ready!